functiondefextractor/condition_checker.py

```diff
@@ -0,0 +1,58 @@
+"""This file contains a function required for patterns in the extracted method/function definitions
```

Comment:
would you like to consider a standard pattern across files to add header
"""Koninklijke Philips N.V., 2019 - 2020. All rights reserved."""

Reply:
Yes. added header in all files.